lms/views/helpers/_via.py

```diff
@@ -45,3 +55,37 @@ def via_url(  # pylint:disable=too-many-arguments
         headers=headers,
         query=query,
     )
+
+
+def via_video_url(
+    request, canonical_url: str, download_url: str, transcript_url: str
+) -> str:
```

Comment:
I think it will probably make sense to move this logic into a method of `ViaClient`. However that will need a separate entry point than `url_for` as for videos there are additional non-optional arguments besides the content URL.